lib/src/model/product.dart

```diff
@@ -925,4 +943,75 @@ class Product extends JsonObject {
     }
     _nutriments = nutriments;
   }
+
+  /// Returns the best version of a product name.
+  ///
+  /// cf. openfoodfacts-server/lib/ProductOpener/Products.pm
+  String getBestProductName(final OpenFoodFactsLanguage language) {
```

Comment:
I would prefer to return a `null` value if there is no match; otherwise you force checking an empty string